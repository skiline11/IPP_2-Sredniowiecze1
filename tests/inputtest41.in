INIT 11 11 1 1 1 1 10
INIT 11 11 2 1 1 1 10
MOVE 2 1 2 2
END_TURN
END_TURN
MOVE 2 2 2 3
END_TURN
END_TURN
MOVE 2 3 2 4
END_TURN
END_TURN
MOVE 2 4 2 5
END_TURN
END_TURN
MOVE 2 6 2 7
END_TURN
END_TURN
MOVE 2 7 2 8
END_TURN
END_TURN
MOVE 2 8 2 9
END_TURN
PRODUCE_PEASANT 2 10 2 9
END_TURN
