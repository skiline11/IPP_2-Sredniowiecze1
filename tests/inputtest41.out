KCRR......
..........
..........
..........
..........
..........
..........
..........
..........
kcrr......

KCRR......
..........
..........
..........
..........
..........
..........
..........
..........
kcrr......

K.RR......
.C........
..........
..........
..........
..........
..........
..........
..........
kcrr......

K.RR......
..........
.C........
..........
..........
..........
..........
..........
..........
kcrr......

K.RR......
..........
..........
.C........
..........
..........
..........
..........
..........
kcrr......

K.RR......
..........
..........
..........
.C........
..........
..........
..........
..........
kcrr......

